src/Beberlei/Metrics/Collector/Graphite.php

```diff
@@ -101,7 +103,12 @@ public function flush()
         $this->data = array();
     }
 
-    public function push($stat, $value, $time = null)
+    /**
+     * @param string $stat
+     * @param string $value
+     * @param int|null $time
+     */
+    private function pushStat($stat, $value, $time = null)
```

Comment:
@lyrixx if you want to deprecate it, introducing a new private method is necessary to avoid using the deprecated API internally (allowing to trigger the deprecation notice).

But the decision to deprecate this API should indeed not belong to this PR. As I said previously, @eexit should have split this PR into many ones:

- one fixing the InfluxDB bug (which could already have been merged)
- one improving the phpdoc
- one introducing the GaugeableCollector interface
- one fixing coding standards

this would make the review much easier, as it would make each PR much smaller. And it would also be much easier to take decision on each of the submitted change, instead of having to take a decision about all of them or none.